### ISSUE TITLE
Calibre Crashing

### ISSUE THREAD

**Issue by artdepthunter** [apprenticeharper/DeDRM_tools#2376]
### DeDRM: `7.2.1`

I am trying to add the Kindle serial number to the plugin, but every time I click on the DeDRM plugin "Customize Plugin" button within Calibre Preferences/Plugins, the application crashes. I can't share a debug report, since the application is crashing, but happy to share the application crash report for MacOS, if helpful.

**Comment by ElleKayEm** [comment]
That version of DeDRM is too old for this version of calibre.  Update to DeDRM 10.0.9: https://github.com/noDRM/DeDRM_tools/releases/tag/v10.0.9

**Comment by artdepthunter** [comment]
Do you have a plugin to remove DRM from Apple EPUBs?

**Comment by ElleKayEm** [comment]
No.  There may be commercial programs but no idea if there are or if they actually work.

**Comment by artdepthunter** [comment]
Thanks for the reply.
